src/Symfony/Component/Console/Formatter/OutputFormatterInterface.php

```diff
@@ -62,8 +62,9 @@ public function getStyle($name);
      * Formats a message according to the given styles.
      *
      * @param string $message The message to style
+     * @param int    $width   The message width
      *
      * @return string The styled message
      */
-    public function format($message);
+    public function format($message/*, $width = 0*/);
```

Comment:
Changing an interface like that is a hard BC break. Any one implementing it without extending `OutputFormatter` will get no deprecation warning.